src/worker.jl

```diff
@@ -2,6 +2,19 @@ function worker_init(f::File, options::Dict)
     project = WorkerSetup.LOADER_ENV[]
     return lock(WorkerSetup.WORKER_SETUP_LOCK) do
         return quote
+            # issue #192
+            # Malt itself uses a new task for each `remote_eval` and because of this, random number streams
+            # are not consistent across runs even if seeded, as each task introduces a new state for its
+            # task-local RNG. As a workaround, we feed all `remote_eval` requests through these channels, such
+            # that the task executing code is always the same.
+            const stable_execution_task_channel_out = Channel()
+            const stable_execution_task_channel_in = Channel() do chan
+                for expr in chan
+                    result = Core.eval(Main, expr)
```

Comment:
I think QuartoNotebookWorker has its own error handling, but you may want to catch errors from this call to prevent future `put!(stable_execution_task_channel_in, ...)` to fail after an exception.

Reply:
Hm yes I might have to think about what to do in this case, but for normal notebook usage, barring bugs in QuartoNotebookRunner, the error handling inside the passed expressions should be sufficient (as indicated by the test suite passing). Of course it would be nicer if in case of bugs, the behavior was still somewhat robust